src/Backups/BackupsWorker.cpp

```diff
@@ -305,8 +305,8 @@ class BackupsWorker::ThreadPools
 
 BackupsWorker::BackupsWorker(ContextMutablePtr global_context, size_t num_backup_threads, size_t num_restore_threads)
     : thread_pools(std::make_unique<ThreadPools>(num_backup_threads, num_restore_threads))
-    , allow_concurrent_backups(global_context->getConfigRef().getBool("backups.allow_concurrent_backups", true))
-    , allow_concurrent_restores(global_context->getConfigRef().getBool("backups.allow_concurrent_restores", true))
+    , allow_concurrent_backups(global_context->getConfigRef().getBool("backups.allow_concurrent_backups", false))
+    , allow_concurrent_restores(global_context->getConfigRef().getBool("backups.allow_concurrent_restores", false))
```

Comment:
It isn't that easy because some tests need to be changed as well.

Reply:
Yeah I'll fix the tests. Wanted to first have all the tests run and then fix the failures.